src/sinks/sematext/metrics.rs

```diff
@@ -176,6 +177,21 @@ impl Service<Vec<Metric>> for SematextMetricsService {
     }
 }
 
+struct SematextMetricNormalize;
+
+impl MetricNormalize for SematextMetricNormalize {
+    fn apply_state(state: &mut MetricSet, metric: Metric) -> Option<Metric> {
+        match &metric.data.value {
+            MetricValue::Gauge { .. } => state.make_absolute(metric),
+            MetricValue::Counter { .. } => state.make_incremental(metric),
+            _ => {
+                emit!(SematextMetricsInvalidMetricReceived { metric: &metric });
```

Comment:
Why are the other implementations of `MetricNormalize` able to handle this case, what's the difference here?

Reply:
The other types of metrics are invalid for Sematext. By dropping them in normalize, we don't have to handle them in the buffer or when creating the request. That logic was moved from `fn encode_events`.